src/Microsoft.DotNet.ProjectJsonMigration/Microsoft.DotNet.Internal.ProjectModel/ProjectReader.cs

```diff
@@ -850,11 +863,132 @@ private static void AddProjectFilesCollectionDiagnostics(JObject rawProject, Pro
                     ErrorCodes.DOTNET1015,
```

Comment:
I have tests, but I don't know if it's from the console or the migration report...